Dockerfile

```diff
@@ -0,0 +1,77 @@
+###############################################################################
+# Backend building stage                                                      #
+###############################################################################
+FROM rust:1.66 as build-backend
+
+# Install dependencies
+RUN apt update
+RUN apt install -y gcc-arm-linux-gnueabihf
+
+# Create a new empty project for the backend
+RUN USER=root cargo new --bin backend
+WORKDIR /backend
+
+# Create cargo config for setting the linker command
+RUN mkdir ./.cargo
+RUN echo "[target.armv7-unknown-linux-gnueabihf]\nlinker = \"arm-linux-gnueabihf-gcc\"" > ./.cargo/config
```

Comment:
or these configs.

Reply:
This binds the previously installed package at line 8 as the linker for cargo to use.